### PR TITLE
Core Alert Fields

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Creating Core Alert Fields pack with new 18 fields.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Action.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
...are: Detected, Detected (Allowed The Session), Detected (Download), Detected (Forward), Detected (Post Detected), Detected (Prompt Allow), Detected (Raised An Alert), Detected (Reported), Detected (Scanned), Detected (Sinkhole), Detected (Syncookie Sent), Detected (Wildfire Upload Failure), Detected (Wildfire Upload Success), Detected (Wildfire Upload Skip), Detected (XDR Managed Threat Hunting), Prevented (Block), Prevented (Blocked), Prevented (Block-Override), Prevented (Blocked The URL), Prevented (Blocked The IP), Prevented (Continue), Prevented (Denied The Session), Prevented (Dropped All Packets), Prevented (Dropped The Session), Prevented (Dropped The Session And Sent a TCP Reset), Prevented (Dropped The Packet), Prevented (Override), Prevented (Override-Lockout), Prevented (Post Detected), Prevented (Prompt Block), Prevented (Random-Drop), Prevented (Silently Dropped The Session With An ICMP Unreachable Message To The Host Or Application), Prevented (Terminated The Session And Sent a TCP Reset To Both Sides Of The Connection), Prevented (Terminated The Session And Sent a TCP Reset To The Client), Prevented (Terminated The Session And Sent a TCP Reset To The Server), N/A.",
```
Comment (reutshal) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Action.json:7]: Looks super long to me, @michalgold WDYT?

File: Packs/CoreAlertFields/IncidentFields/incidentfield-File_SHA256.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "6.5.0",
+	"x2_fields": "action_file_sha256",
+    "Aliases": [
+        {
+          "cliName": "sha256",
```
Comment (reutshal) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-File_SHA256.json:34]: Add: "marketplaces - xsoar" to the field

File: Packs/CoreAlertFields/IncidentFields/incidentfield-User_name.json [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "6.5.0",
+	"x2_fields": "actor_effective_username"ת
```
Comment (reutshal) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-User_name.json:31]:
```suggestion
	"x2_fields": "actor_effective_username",
```

**Review by reutshal — CHANGES_REQUESTED** [review]
Please add all aliases fields to marketplace xsoar only

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Action.json (hunk 1 above)
Comment (reutshal) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Action.json:7]:
```suggestion
    "description": "Action taken by the alert sensor, either Detected or Prevented with action status displayed in parenthesis. Options are: Detected, Detected (Allowed The Session), Detected (Download), Detected (Forward), Detected (Post Detected), Detected (Prompt Allow), Detected (Raised An Alert), Detected (Reported), Detected (Scanned), Detected (Sinkhole), Detected (Syncookie Sent), Detected (Wildfire Upload Failure), Detected (Wildfire Upload Success), Detected (Wildfire Upload Skip), Detected (XDR Managed Threat Hunting), Prevented (Block), Prevented (Blocked), Prevented (Block-Override), Prevented (Blocked The URL), Prevented (Blocked The IP), Prevented (Continue), Prevented (Denied The Session), Prevented (Dropped All Packets), Prevented (Dropped The Session), Prevented (Dropped The Session And Sent a TCP Reset), Prevented (Dropped The Packet), Prevented (Override), Prevented (Override-Lockout), Prevented (Post Detected), Prevented (Prompt Block), Prevented (Random-Drop), Prevented (Silently Dropped The Session With An ICMP Unreachable Message To The Host Or Application), Prevented (Terminated The Session And Sent a TCP Reset To Both Sides Of The Connection), Prevented (Terminated The Session And Sent a TCP Reset To The Client), Prevented (Terminated The Session And Sent a TCP Reset To The Server).",
```

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "6.5.0",
+	"x2_fields": "actor_process_image_name",
+    "Aliases": [
+        {
+          "cliName": "processname",
```
Comment (reutshal) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json:34]: Add xsoar marketplace

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Initiator_path.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "6.5.0",
+	"x2_fields": "actor_process_image_path",
+    "Aliases": [
+        {
+          "cliName": "processpath",
```
Comment (reutshal) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Initiator_path.json:34]: Add xsoar marketplace

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Local_Port.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        {
+          "cliName": "fireeyenxeventvictimport",
+          "type": "shortText"
+        },
+        {
+          "cliName": "sourceport",
+          "type": "shortText"
+        },
+        {
+        "cliName": "srcports",
```
Comment (reutshal) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Local_Port.json:46]: Add xsoar marketplace and remove aliases

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Remote_Port.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "6.5.0",
+	"x2_fields": "action_remote_port",
+    "Aliases": [
+        {
+          "cliName": "dstports",
```
Comment (reutshal) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Remote_Port.json:34]: remove aliases and xsoar only

**Review by reutshal — CHANGES_REQUESTED** [review]

**Review by reutshal — APPROVED** [review]

**Comment by esharf** [comment]
@tomer-pan @reutshal 
```
[2022-02-01 00:33:52] - [MainThread] - [ERROR] - Failed in formatting Malware pack metadata. Additional Info: New mandatory dependencies ['CoreAlertFields'] were found in the core pack Malware
Traceback (most recent call last):
  File "/builds/xsoar/content/Tests/Marketplace/marketplace_services.py", line 1914, in format_metadata
    self.set_pack_dependencies(packs_dependencies_mapping)
  File "/builds/xsoar/content/Tests/Marketplace/marketplace_services.py", line 2010, in set_pack_dependencies
    raise Exception(f'New mandatory dependencies {mandatory_dependencies} were '
Exception: New mandatory dependencies ['CoreAlertFields'] were found in the core pack Malware
```
nightly is failing because of this error.
can you check it?

**Comment by tomer-pan** [comment]
> @tomer-pan @reutshal
> 
> ```
> [2022-02-01 00:33:52] - [MainThread] - [ERROR] - Failed in formatting Malware pack metadata. Additional Info: New mandatory dependencies ['CoreAlertFields'] were found in the core pack Malware
> Traceback (most recent call last):
>   File "/builds/xsoar/content/Tests/Marketplace/marketplace_services.py", line 1914, in format_metadata
>     self.set_pack_dependencies(packs_dependencies_mapping)
>   File "/builds/xsoar/content/Tests/Marketplace/marketplace_services.py", line 2010, in set_pack_dependencies
>     raise Exception(f'New mandatory dependencies {mandatory_dependencies} were '
> Exception: New mandatory dependencies ['CoreAlertFields'] were found in the core pack Malware
> ```
> 
> nightly is failing because of this error. can you check it?

@reutshal I believe it's related to the same thing we were talking about where the sdk should add the dependencies to the metadata of the packs with the fields that their aliases are in the new fields in the Core Alert Fields pack.

**Pull request merged**